### PR TITLE
Add README.md to integration test crates

### PR DESCRIPTION
**Pull request by johnsonw**
- Add README.md to docker-based integration tests
- Add README.md to rpm-based integration tests
- Add README.md to iml-system-test-utils discussing test architecture

Signed-off-by: johnsonw <wjohnson@whamcloud.com>

---
This change is [Reviewable](https://reviewable.io/reviews/whamcloud/integrated-manager-for-lustre/2091)

### CODE REVIEW

File: iml-system-rpm-tests/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# IML System RPM Tests
+
+The system level RPM integration tests focus on the creation of various forms of RPM-based lustre filesystems. Vagrant is used to launch the following virtual machines:
+
+1. iscsi - Used for filesystem devices
+1. adm - The manager node
+1. mds[1,2] - The MDS nodes
+1. oss[1,2] - The OSS nodes
```
Comment (jgrund) [inline iml-system-rpm-tests/README.md:8]: clients

File: iml-system-rpm-tests/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+## Installation
+
+Install the following software:
+
+1. cargo
+1. rust
+1. sos
```
Comment (jgrund) [inline iml-system-rpm-tests/README.md:16]: Is sos needed on the host?

Comment (johnsonw) [reply iml-system-rpm-tests/README.md:16]: That's a good question. I took this from the jenkins pipeline but thinking about it more I don't think it is. Maybe I should remove it from the pipeline as well.

File: iml-system-docker-tests/README.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+1. mds[1,2] - The MDS nodes
+1. oss[1,2] - The OSS nodes
+
+## Installation
+
+Install the following software:
+
+1. cargo
+1. rust
+1. sos
```
[the remark this answers is not in the document]
Comment (johnsonw) [reply iml-system-docker-tests/README.md:16]: removed

File: iml-system-rpm-tests/README.md (hunk 2 above)
Comment (jgrund) [reply iml-system-rpm-tests/README.md:16]: Since everything runs in the VM, I think we don't need it on the host.

**Pull request merged**